docs/rst/user_manual/chart_panel_index.rst

```diff
@@ -7,13 +7,42 @@
 Main Panel
 ##########
 
-In the central panel or *chart panel*, is where the data of the entities will be displayed.
-The main feature of the *Fast DDS Monitor* application is to graphically display the data that is being monitored.
+In the central panel, there is a tan section that allows multiple views, including a collapsed menu that reports the
+possible problems that have been detected on the DDS entities.
+
+The main feature of the *Fast DDS Monitor* application is to graphically display the data that is being monitored in
+the *Chartbox* or *Chart view*.
```

Comment:
See review comments in Chartbox section from layout.rst. Here I would only use ChartView.